app/controllers/testOnlyDoNotUseInAppConf/TestFeatureSwitchManagerController.scala

```diff
@@ -31,59 +32,101 @@ class TestFeatureSwitchManagerController @Inject()(
                                                     adminFeatureSwitchConnector: PensionAdministratorFeatureSwitchConnectorImpl
                                                   )(implicit val ec: ExecutionContext) extends FrontendController {
 
```

Comment:
Logic fails when the toggle state is false

Reply:
Sorted